### PR TITLE
752 - Fixed: Suggestion to add -v remains even if you use -v

### PR DESCRIPTION
**Pull request by BrnoPCmaniak**
closes #752
https://pulp.plan.io/issues/752

### CODE REVIEW

**Comment by asmacdo** [comment]
I would like to see this a little DRYer. Perhaps a new function?

**Comment by BrnoPCmaniak** [comment]
ok test

File: client_lib/pulp/client/extensions/exceptions.py [hunk 1]
```diff
@@ -126,8 +126,14 @@ def handle_bad_request(self, e):
                     'provided the following error message:')
             self.prompt.render_failure_message(msg)
 
-            self.prompt.render_failure_message('   %s' % e.error_message)
-            msg = _('More information may be found using the -v flag.')
+            msg = '   %s' % e.error_message
```
Comment (asmacdo) [inline client_lib/pulp/client/extensions/exceptions.py]: Can this block be handled by the new function as well?

Comment (BrnoPCmaniak) [inline client_lib/pulp/client/extensions/exceptions.py]: I don't think so there would be an extra empty ``self.prompt.render_failure_message(msg)``

Comment (BrnoPCmaniak) [inline client_lib/pulp/client/extensions/exceptions.py]: I think I just came up with DRYer solution.

File: client_lib/test/unit/client/extensions/test_exceptions.py [hunk 2]
```diff
@@ -416,3 +416,22 @@ def test_certificate_expiration(self):
 
         # Verify
         self.assertEqual(date, 'May  9 12:39:37 2013 GMT')
+
```
Comment (asmacdo) [inline client_lib/test/unit/client/extensions/test_exceptions.py]: I would like to see these tests use the Mock library to so you can test all the options. That will also not depend on the state of logging.NOTSET.

File: client_lib/pulp/client/extensions/exceptions.py (hunk 1 above)
Comment (asmacdo) [inline client_lib/pulp/client/extensions/exceptions.py]: nice!

File: client_lib/test/unit/client/extensions/test_exceptions.py (hunk 2 above)
Comment (asmacdo) [inline client_lib/test/unit/client/extensions/test_exceptions.py]: Ping me on irc if you want me to help you get started with that.

Comment (BrnoPCmaniak) [inline client_lib/test/unit/client/extensions/test_exceptions.py]: I updated them to use Mock so all options are tested.

Comment (asmacdo) [inline client_lib/test/unit/client/extensions/test_exceptions.py]: Nice work.

**Comment by asmacdo** [comment]
That test failure seems to be happening sometimes, and is not related to this PR. Feel free to run the tests again or merge.

**Comment by BrnoPCmaniak** [comment]
ok test

**Pull request merged**